### DISCUSSION TITLE
Exclude files

### DISCUSSION THREAD

**Discussion by rubiin** [twpayne/chezmoi#3794] [General]
Is there a way to skip updating encrypted files when using chezmoi update?

**Comment by twpayne** [comment]
Yes, use the [`--exclude` flag](https://www.chezmoi.io/reference/command-line-flags/common/#-x-exclude-types):

```console
$ chezmoi update --exclude=encrypted
```

You asked the same question in https://github.com/twpayne/chezmoi/issues/3747#issuecomment-2143375567.

**Reply by rubiin** [reply]
yeah . Found out about it after posting lol
